pkgs/development/python-modules/opencensus/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, python
+, unittestCheckHook
+, google-api-core
+}:
+
+let
+  opencensus-context = buildPythonPackage rec {
+    pname = "opencensus-context";
+    version = "0.1.3";
+
+    checkInputs = [ unittestCheckHook ];
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "a03108c3c10d8c80bb5ddf5c8a1f033161fa61972a9917f9b9b3a18517f0088c";
+    };
+  };
+in
+buildPythonPackage rec {
+  pname = "opencensus";
+  version = "0.11.0";
+
+  checkInputs = [ unittestCheckHook ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "026216abab89d94d850492f3dc65950055ec4f84115fa6c7beaffba44a346e42";
+  };
+
+  buildInputs = [
+    # opencensus-context is embedded in opencensus
+    opencensus-context
+  ];
```

Comment:
Why is it then in buildInputs? Is this only required to build the final package?

Reply:
Yes, kind of.
The relation between `opencensus` and `opencensus-context` is wired. They both will be installed to `site-packages/opencensus`, and opencensus will relatively import contents from `opencensus-context`. so `opencensus-context` is actually a buildinput that needs to be copied/linked to `opencensus`.